R/Utils.R

```diff
@@ -262,6 +277,11 @@ is_parameterised <- function(ps) {
   rJava::.jcall(md, "I", "getParameterCount") > 0
 }
 
+num_parameters <- function(ps) {
```

Comment:
`num_parameters` returns the number, and I use a numerical comparison in `dbSendQuery`; `is_parameterised` is only yes/no, I wanted more. It's one of several internal tests to prevent difficult-to-track-down errors (that I hit during my own testing).